docs/reports/tootsie-retro.md

```diff
@@ -0,0 +1,416 @@
+# Tootsie 8B Retrospective
+
+This is a retrospective on our first generation Marin 8B run, codenamed "Tootsie."
+We include details on the data mix, hyperparameters, and other details. We also include observations during the run and
+document some of the mistakes we made and lessons learned.
+
+This document was not created during the run, but rather after the fact. Some details may be missing or incorrect,
+but we have tried to be as accurate as possible.
+
+In addition, the [actual code](https://github.com/marin-community/marin/blob/852c53f9741b233549daf9f1649fe88c9c5a170c/experiments/tootsie/exp600_tootsie.py)
```

Comment:
I think tootsie should everything up to the sft'ed deeper-starling...so we need to include the other files